engine/tools/src/gdc/main.cpp

```diff
@@ -32,6 +32,8 @@
 #include <hid/hid.h>
 #include <input/input_ddf.h>
 
+extern "C" void dmExportedSymbols();
```

Comment:
In this release, we've removed the "magic" registering of exported symbols (that used c++ global constructors), in favor of actually calling this generated function.

It will help us support extensions on more languages+platforms.